Resources/Prototypes/Entities/Mobs/NPCs/xeno.yml

```diff
@@ -57,14 +57,16 @@
         - MobMask
         layer:
         - MobLayer
-  - type: MobState
-    allowedStates:
-    - Alive
-    - Dead
+      actions:
+      Critical:
+      - ActionCritSuccumb
+      - ActionCritFakeDeath
+      - ActionCritLastWords
```

Comment:
This all is invalid syntax

Reply:
I'm confused how is it invalid syntax, I tried to fix it but- it doesn't seem to have worked